Software/Backend/WebApi/Controllers/JobController.cs

```diff
@@ -34,9 +36,31 @@ public ActionResult<AddJobResponse> AddJob([FromBody] AddJobRequest request)
 
         if (!string.IsNullOrEmpty(response.Error))
         {
+
             return BadRequest(response);
         }
 
         return response;
     }
+
+    // GET: /jobsForCurrentUser
+    [HttpGet("jobsForCurrentUser")]
+    [Authorize]
+    public ActionResult<GetJobsForCurrentUserResponse> GetJobsForCurrentUser()
+    {
+        var userIdFromJwt = HttpContext.Items["UserId"] as int?;
+
+        if (userIdFromJwt == null)
+        {
+            return Unauthorized(new GetJobsForCurrentUserResponse()
+            {
+                Error = "Ne mo�ete pristupiti tom resursu!"
+            });
+        }
+
+        var jobs = _jobService.GetJobsForCurrentUser(userIdFromJwt.Value);
+        
+        return jobs;
+    }
+
```

Comment:
Ovaj naziv metode "/jobsForCurrentUser" je možda malo ne-specifičan, bolji bi bio npr. "/jobs/search" ili tako nešto? U redu si nazvao model JobSearchModel.